System test `test_end_all_reviews` failing

Output: 

```
================================================= test session starts ==================================================
platform linux -- Python 3.8.10, pytest-7.0.1, pluggy-1.0.0
rootdir: /home/adam/code/dd2480/repobee
plugins: cov-3.0.0, mock-3.7.0, repobee-3.8.0.dev0
collected 25 items

test_gitlab_system.py .....................F...                                                                  [100%]

======================================================= FAILURES =======================================================
_________________________________________ TestEndReviews.test_end_all_reviews __________________________________________

self = <test_gitlab_system.TestEndReviews object at 0x7fc27dc403a0>
with_reviews = ('task-3', [StudentTeam(members=[], name='simon-task-3-review'), StudentTeam(members=[], name='alice-task-3-review'), StudentTeam(members=[], name='eve-task-3-review')])
tmpdir = local('/tmp/pytest-of-adam/pytest-3/test_end_all_reviews0')

    def test_end_all_reviews(self, with_reviews, tmpdir):
        assignment_name, review_teams = with_reviews
        command = " ".join(
            [
                *repobee_plug.cli.CoreCommand.reviews.end.as_name_tuple(),
                *BASE_ARGS,
                "-a",
                assignment_name,
                *STUDENTS_ARG,
            ]
        )

        run_repobee(command, workdir=tmpdir, plugins=[_repobee.ext.gitlab])

        def assert_no_actual_groups(expected, actual):
            assert not actual

        # student teams should still exist
        assert_on_groups(STUDENT_TEAMS)
        # review teams should not
>       assert_on_groups(
            review_teams, all_groups_assertion=assert_no_actual_groups
        )

test_gitlab_system.py:682:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
_helpers/asserts.py:92: in assert_on_groups
    all_groups_assertion(sorted_teams, sorted_groups)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

expected = [StudentTeam(members=[], name='alice-task-3-review'), StudentTeam(members=[], name='eve-task-3-review'), StudentTeam(members=[], name='simon-task-3-review')]
actual = [<Group id:110>, <Group id:111>, <Group id:109>]

    def assert_no_actual_groups(expected, actual):
>       assert not actual
E       assert not [<Group id:110>, <Group id:111>, <Group id:109>]

test_gitlab_system.py:677: AssertionError
```

--- a/.github/workflows/tests.yml
+++ b/.github/workflows/tests.yml
@@ -68,8 +68,8 @@ jobs:
 
   gitlab-system-tests:
     runs-on: ubuntu-latest
-    if: ${{ github.event_name == 'create' \
-      || github.event_name == 'schedule' \
+    if: ${{ github.event_name == 'create'
+      || github.event_name == 'schedule'
       || (github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'gitlab')) }}
     steps:
     - uses: actions/checkout@v3

--- a/.github/workflows/tests.yml
+++ b/.github/workflows/tests.yml
@@ -68,7 +68,9 @@ jobs:
 
   gitlab-system-tests:
     runs-on: ubuntu-latest
-    if: ${{ github.event_name == 'create' || gitlab.event_name == 'schedule' || (github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'gitlab')) }}
+    if: ${{ github.event_name == 'create' \
+      || github.event_name == 'schedule' \
+      || (github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'gitlab')) }}
     steps:
     - uses: actions/checkout@v3
       with:

--- a/.github/workflows/tests.yml
+++ b/.github/workflows/tests.yml
@@ -68,7 +68,9 @@ jobs:
 
   gitlab-system-tests:
     runs-on: ubuntu-latest
-    if: ${{ github.event_name == 'create' || (github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'gitlab')) }}
+    if: ${{ github.event_name == 'create'
+      || gitlab.event_name == 'schedule'
+      || (github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'gitlab')) }}
     steps:
     - uses: actions/checkout@v3
       with:

--- a/.github/workflows/tests.yml
+++ b/.github/workflows/tests.yml
@@ -68,8 +68,8 @@ jobs:
 
   gitlab-system-tests:
     runs-on: ubuntu-latest
-    if: ${{ github.event_name == 'create'
-      || gitlab.event_name == 'schedule'
+    if: ${{ github.event_name == 'create' \
+      || gitlab.event_name == 'schedule' \
       || (github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'gitlab')) }}
     steps:
     - uses: actions/checkout@v3

--- a/.github/workflows/tests.yml
+++ b/.github/workflows/tests.yml
@@ -68,9 +68,7 @@ jobs:
 
   gitlab-system-tests:
     runs-on: ubuntu-latest
-    if: ${{ github.event_name == 'create' \
-      || gitlab.event_name == 'schedule' \
-      || (github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'gitlab')) }}
+    if: ${{ github.event_name == 'create' || gitlab.event_name == 'schedule' || (github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'gitlab')) }}
     steps:
     - uses: actions/checkout@v3
       with:

